Apps/Common/SIMSolverTS.h

```diff
@@ -128,11 +128,10 @@ template<class T1> class SIMSolverTS : public SIMSolver<T1>
             return 2;
       }
     }
-
-    return 0;
```

Comment:
What? Remove the return value for solveProblem??? Wouldn't this only yield a compiler warning, or what do I not see here...